__tests__/habit-page.test.tsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * @jest-environment jsdom
+ */
+import { render, screen } from "@testing-library/react";
+import HabitPage from "../app/habit/habit-page";
+import { useSession } from "next-auth/react";
+import useSwr from "swr";
+
+jest.mock("next-auth/react");
+jest.mock("swr");
+
+it("HabitPage has the 'Habits' title", () => {
+  (useSession as jest.Mock).mockReturnValue({
+    data: {
+      accessToken: "test",
+    },
+    status: "authenticated",
+  });
+
+  (useSwr as jest.Mock).mockReturnValue({
+    data: [{ name: "habit1" }, { name: "habit2" }],
+    error: null,
+    isLoading: false,
+    mutate: () => {},
+  });
+
+  render(<HabitPage currentHabit="test" setCurrentHabit={() => {}} />);
+  expect(screen.getByRole("heading").textContent).toBe("Habits");
```

Comment:
please also check if changing this line to the below one still works as expected:
```
expect(screen.getByRole("heading")).toHaveTextContent("Habits");
```

After changing the line locally:
1. make sure that your IDE doesn't show an error for `toHaveTextContent`
2. run tests with `npm run test` and make sure that the test passes